modules/data.remote/R/call_MODIS.R

```diff
@@ -14,6 +14,8 @@
 ##' @param band_qc string value for which quality control band, or use "NA" if you do not know or do not need QC information (optional)
 ##' @param band_sd string value for which standard deviation band, or use "NA" if you do not know or do not need StdDev information (optional)
 ##' @param package_method string value to inform function of which package method to use to download modis data. Either "MODISTools" or "reticulate" (optional)
+##' @param siteID string value of a PEcAn site ID. Currently only used for output filename.
```

Comment:
@bailsofhay is this necessary? Seems very PEcAn/BETY-centric but not sure if we want to have something like this for a user who may not have a site ID.  I would instead think it may be better to allow the user to define the output filename

Reply:
also after any changes to the docs you need to remake the documentation

Reply:
I will re-do the doc soon. the siteID thing was for something else I was playing with. it's not needed anymore so I'm removing it all together

Reply:
OHHH I know what it was for, this was when I was getting an error that would break the function from the site query (we used suppressWarnings() to fix this). I needed it to output the data in a way that made sense.